packages/frontend/package.json

```diff
@@ -83,7 +83,7 @@
         "react-json-view": "^1.21.3",
         "react-redux": "^9.1.2",
         "react-resizable-panels": "^2.1.2",
-        "react-router-dom": "5.3.4",
+        "react-router-dom": "6.0.0",
```

Comment:
Breaking changes need to be addressed first.
Read more: https://reactrouter.com/en/main/upgrading/v5